src/documentation/pages/components/toolbar/_examples/_base.vue

```diff
@@ -0,0 +1,585 @@
+<template lang="pug">
+  <ComponentExample title="Base" id="base" :tabs="exampleTabs">
```

Comment:
It seems you are missing icons:

![image](https://user-images.githubusercontent.com/86958774/156769092-68468f7c-2549-4332-9f7a-a261aede0f8b.png)

Also double check the code snippets.

Reply:
Oh true I'm not adding those other icons because they aren't interactive, but i do leave them all like they are in the old documentation in the non-interactive examples.

Reply:
It's also why the code snippets don't match exactly